source/parser/memory-allocator.cpp

```diff
@@ -86,17 +85,17 @@ std::size_t MemoryAllocator::calculatePositions() {
   // rounding up to align.
   for (auto& i : _sections) {
     auto sectionAlign = i._definition.sectionAlignment;
-    auto aligned =
-        Utility::discreteCeiling(position, sectionAlign) * sectionAlign;
+    auto aligned = Utility::discreteCeiling(position, sectionAlign);
     i._currentPosition = aligned;
     position = aligned + i._currentSize;
   }
   return position;
 }
 
 // Accessor functions...
-MemoryAllocator::MemorySection& MemoryAllocator::operator[](std::string name) {
-  assert::that(_nameToSection.find(name) != _nameToSection.end());
+MemoryAllocator::MemorySection& MemoryAllocator::
```

Comment:
Don't really like the breaking here. Prefer breaking the return type if it won't fit:
```
+MemoryAllocator::MemorySection&
MemoryAllocator::operator[](const std::string& name) {
```